BlazorUI/Data/WeatherForecastService.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BlazorUI.Data
+{
+    public class WeatherForecastService
+    {
+        private static readonly string[] Summaries = new[]
+        {
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
+
+        public Task<WeatherForecast[]> GetForecastAsync(DateTime startDate)
+        {
+            var rng = new Random();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make sure that using this pseudorandom number generator is safe here.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)